comp/core/gui/guiimpl/views/templates/index.tmpl

```diff
@@ -61,12 +61,18 @@
     <div id="title">Datadog Agent Manager</div>
     <div id="agent_info">
       <div id="restart_status">Restart Agent to apply changes.</div>
-      <div id="agent_status">Connected <br>to Agent</div>
+      <div id="agent_status" class="disconnected">Connected <br>to Agent</div>
```

Comment:
Nit: It is weird that the dic class is disconnected and the text is `Connected` 😄

Reply:
We are not adding any special css to the disconnected and connected classes, right? 
If that is the case do we need them?

Reply:
Good catch ! I updated it in dd3e20511111bb96edaf7395d30ab0ccae8cee24
I did add some CSS classes:
```
.connected {
  background: linear-gradient(to bottom, #89c403 5%, #77a809 100%);
  background-color: #89c403;
  border: 1px solid #74b807;
  text-shadow: 0px 1px 0px #528009;
  left: -150px;
}
.disconnected {
  background: linear-gradient(to bottom, #c62d1f 5%, #f24437 100%);
  background-color: #c62d1f;
  border: 1px solid #d02718;
  text-shadow: 0px 1px 0px #810e05;
  left: -180px;
}
```
It basically set the background color to red/green. Before it was set dynamically in the javascript script with jQuerry.